### DISCUSSION TITLE
stuck on test page after ubuntu update

### DISCUSSION THREAD

**Discussion by cbrownrome** [ampache/ampache#3912] [Q&A] [answered]
Ampache WebUI remains stuck on test page with PHP errors and MySQL database connection issues

**Description:**
I am encountering an issue with the Ampache WebUI: the page remains stuck on the test page (see screenshot). This, I presume, occurred after an operating system update. Indeed, Ampache was functioning properly before this occurrence, and there were no changes made to the program's installation process.
**Environment:**
Ampache Version: 6.2.1
SQL: MariaDB (SQL)
Operating System: Linux 5.4.0-174-generic #193-Ubuntu
Browser: chrome, firefox
**Screenshot:**
![ae3b711e-0d2e-4256-b04d-7e2e701ce471](https://github.com/ampache/ampache/assets/103494856/faa2d86d-2967-4659-9eb7-2c172ada2ea0)

Thank you for your assistance in resolving this issue.

**Comment by stebe** [comment]
Looks like your version of PHP may have been upgraded.  Make sure that php-intl and php-msql packages are installed.  They may have been removed by the upgrade.  sudo apt install php-intl php-mysql.  Optional packages to install are php-curl php-xml and php-gd

**Reply by cbrownrome** [reply]
all these packages, including optional ones, are already installed   (i.e. php-intl is already at the latest version (2:8.3+94+ubuntu20.04.1+deb.sury.org+2) )

**Reply by stebe** [reply, marked as answer]
This is a long shot but in my case when PHP got updated the configuration files are not transferred to the new version of php.ini.  You may have to manually update the configuration files. 

Is Apache using the correct version of PHP? For example, I'm using php8.2-fpm.

```
a2query  -c php8.2-fpm
php8.2-fpm (enabled by site administrator)

a2query  -c php8.3-fpm
No conf matches php8.3-fpm

etc/apache2/conf-available# a2query  -c
serve-cgi-bin (enabled by maintainer script)
security (enabled by maintainer script)
httpd-proxy (enabled by site administrator)
localized-error-pages (enabled by maintainer script)
backuppc (enabled by maintainer script)
charset (enabled by maintainer script)
php8.2-fpm (enabled by site administrator)
apache2-doc (enabled by maintainer script)
phpmyadmin (enabled by maintainer script)
ttrss (enabled by site administrator)
other-vhosts-access-log (enabled by maintainer script)
javascript-common (enabled by maintainer script)


    <FilesMatch ".+\.ph(?:ar|p|tml)$">
        SetHandler "proxy:unix:/run/php/php8.2-fpm.sock|fcgi://localhost"
    </FilesMatch>

```

**Reply by cbrownrome** [reply]
Thanks. You helped me to solve a lot.
I used  
`sudo apt install php8.3-fpm`
to update, 
then 
`sudo a2enconf php8.3-fpm`  
to enable the new apache configuration
and restarted apache
